src/flaresolverr_service.py

```diff
@@ -287,20 +287,36 @@ def click_verify(driver: WebDriver):
     time.sleep(2)
 
 
```

Comment:
Maybe we should elaborate why we need to find the correct window. I mean, it's clear once it says that the devtools are required to beat the challenge.

By the way, now that the devtools are open by default, how can we test whether all other Cloudflare challenges still work properly? I guess it will work, just mentioning it so someone can think about it.
I didn't check the complete source code right now because I am on mobile

Reply:
I think this repository could have some automated tests to check a dozen pages or so. I checked a few pages for my own needs and a few which I found in the issue comments.

Usually from my experience if it works even for one site solving the captcha - it will work for others as well

Reply:
https://github.com/FlareSolverr/FlareSolverr/blob/master/src/tests.py
https://github.com/FlareSolverr/FlareSolverr/blob/master/src/tests_sites.py